docs/output.md

```diff
@@ -568,135 +685,187 @@ Currently, it contains:
-  - Zip archive containing the FastQC report, tab-delimited data file and plot images
+- `<sample-lane_1>_fastqc.html` and `<sample-lane_2>_fastqc.html`
+  - [FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) report containing quality metrics for your untrimmed raw FastQ files
+- `<sample-lane_1>_fastqc.zip` and `<sample-lane_2>_fastqc.zip`
+  - Zip archive containing the [FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) report, tab-delimited data file and plot images
 
-> **NB:** The `FastQC` plots displayed in the `MultiQC` report shows _untrimmed_ reads.
+> **NB:** The FastQC plots displayed in the [MultiQC](https://multiqc.info/) report shows _untrimmed_ reads.
 > They may contain adapter sequence and potentially regions with low quality.
 
-- `fastqc/`
-  - `*_fastqc.html`: FastQC report containing quality metrics.
-  - `*_fastqc.zip`: Zip archive containing the FastQC report, tab-delimited data file and plot images.
+</details>
 
-#### bamQC
+#### FastP
 
-[Qualimap bamqc](http://qualimap.bioinfo.cipf.es/) reports information for the evaluation of the quality of the provided alignment data.
-In short, the basic statistics of the alignment (number of reads, coverage, GC-content, etc.) are summarized and a number of useful graphs are produced.
+[FastP](https://github.com/OpenGene/fastp) is a tool designed to provide all-in-one preprocessing for FastQ files and as such is used for trimming and splitting. The tool then determines QC metrics for the processed reads.
 
-Plot will show:
+<details markdown="1">
+<summary>Output files for all samples</summary>
 
-- Stats by non-reference allele frequency, depth distribution, stats by quality and per-sample counts, singleton stats, etc.
+**Output directory: `{outdir}/reports/fastp/<sample>`**
 
-For all samples:
+- `<sample-lane>_fastp.html`
+  - report in HTML format
+- `<sample-lane>_fastp.json`
+  - report in JSON format
+- `<sample-lane>_fastp.log`
+  - FastQ log file
 
-**Output directory: `results/Reports/[SAMPLE]/bamQC`**
+</details>
 
-- `VariantCaller_[SAMPLE].bcf.tools.stats.out`
-  - Raw statistics used by `MultiQC`
+#### Mosdepth
+
+[Mosdepth](https://github.com/brentp/mosdepth) reports information for the evaluation of the quality of the provided alignment data.
+In short, the basic statistics of the alignment (number of reads, coverage, GC-content, etc.) are summarized and a number of useful graphs are produced.
+For further reading and documentation see the [Mosdepth](https://github.com/brentp/mosdepth).
+
+Plots will show:
 
-For further reading and documentation see the [Qualimap bamqc manual](http://qualimap.bioinfo.cipf.es/doc_html/analysis.html#id7)
+- cumulative coverage distribution
+- absolute coverage distribution
+- average coverage per contig/chromosome
+
+<details markdown="1">
+<summary>Output files for all samples</summary>
+
+**Output directory: `{outdir}/reports/mosdepth/<sample>`**
+
+- `<sample>.{md,recal}.mosdepth.global.dist.txt`
```

Comment:
Didn't something change here? In case of `--skip_markduplicates` is skipped, there can also be `.sorted`. See here https://github.com/nf-core/sarek/blob/c43ef3376a9acd368a28df3ee490e10e9280da9d/tests/test_skip_markduplicates.yml#L23

Reply:
addressed. Does this also concern the multiqc config?